internal/pkg/teleport/token.go

```diff
@@ -32,21 +32,24 @@ func (t *Teleport) IsTokenValid(ctx context.Context, registerName string, oldTok
 func (t *Teleport) GenerateToken(ctx context.Context, registerName string, tokenType string) (string, error) {
 	var (
 		tokenValidity time.Time
-		tokenRole     tt.SystemRole
+		tokenRoles    []types.SystemRole
 	)
 
 	switch tokenType {
 	case "kube":
 		tokenValidity = time.Now().Add(key.TeleportKubeTokenValidity)
-		tokenRole = tt.RoleKube
+		tokenRoles = []types.SystemRole{types.RoleKube}
+	case "kubeapp":
+		tokenValidity = time.Now().Add(key.TeleportKubeTokenValidity)
+		tokenRoles = []types.SystemRole{types.RoleKube, types.RoleApp}
```

Comment:
I'm new to this - are there cases where we'd ever want to issue only an `app` token? Maybe to have non-kube access for some users, or to separate the management of the app and kube roles?